src/components/ThemeSwitcher.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+
+import { HiOutlineMoon, HiOutlineSun } from 'react-icons/hi';
```

Comment:
```suggestion
import { HiOutlineMoon, HiOutlineSun } from 'react-icons/hi/index';
```
Vite is unable to resolve the file without specifying `index` in-case of `react-icons` package.